reports/meta.tex

```diff
@@ -26,7 +26,9 @@ \subsubsection{Bounded recursion rule}
   \item A general case operator $G$, which invokes a recursive computation of $R$ over a modified parameter given by the operator $\bb$.
 \end{itemize}
 %
-The following needs to hold true, to ensure recursion termination: for every $x\colon a$, there exists a finite sequence $x = v_1, \dots, v_n$, such that
+The following needs to hold true, to ensure recursion termination: for every $x\colon a$, there exists a finite sequence $x = v_1, \dots, v_n$
+\tth{Should we use the same notation $\tup{v_n, \ldots, x}$ for sequences as on Page 10.}
```

Comment:
We could, but I don't like propagating TLA notation into proofs if it can be avoided.